[2024-06-18T12:40:22.154+0400] {processor.py:161} INFO - Started process (PID=45620) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:40:22.155+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:40:22.155+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:40:22.155+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:40:22.157+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:40:22.157+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:40:22.157+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:40:22.157+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:40:22.158+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:40:22.165+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-06-18T12:42:58.038+0400] {processor.py:161} INFO - Started process (PID=46573) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:42:58.040+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:42:58.041+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:42:58.041+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:42:58.043+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:42:58.043+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:42:58.043+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:42:58.043+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:42:58.043+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:42:58.051+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-06-18T12:43:49.604+0400] {processor.py:161} INFO - Started process (PID=46750) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:43:49.605+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:43:49.606+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:43:49.606+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:43:49.609+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:43:49.609+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:43:49.609+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:43:49.609+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:43:49.609+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:43:49.615+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-06-18T12:44:39.860+0400] {processor.py:161} INFO - Started process (PID=46895) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:44:39.863+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:44:39.865+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:44:39.864+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:44:39.867+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:44:39.867+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:44:39.868+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:44:39.868+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:44:39.868+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:44:39.874+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-06-18T12:45:22.353+0400] {processor.py:161} INFO - Started process (PID=47011) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:45:22.355+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:45:22.356+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:45:22.356+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:45:22.358+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:45:22.358+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:45:22.358+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:45:22.358+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:45:22.358+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:45:22.364+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T12:46:05.775+0400] {processor.py:161} INFO - Started process (PID=47105) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:46:05.779+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:46:05.781+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:46:05.780+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:46:05.784+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:46:05.784+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:46:05.784+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:46:05.784+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:46:05.785+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:46:05.795+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-06-18T12:47:04.306+0400] {processor.py:161} INFO - Started process (PID=47868) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:47:04.309+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:47:04.310+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:47:04.310+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:47:04.312+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:47:04.312+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:47:04.312+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:47:04.312+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:47:04.313+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:47:04.318+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-06-18T12:48:01.444+0400] {processor.py:161} INFO - Started process (PID=48186) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:48:01.447+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:48:01.449+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:48:01.449+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:48:01.452+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:48:01.451+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:48:01.452+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:48:01.452+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:48:01.452+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:48:01.459+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-06-18T12:48:52.354+0400] {processor.py:161} INFO - Started process (PID=48447) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:48:52.356+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:48:52.356+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:48:52.356+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:48:52.358+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:48:52.358+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:48:52.358+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:48:52.358+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:48:52.358+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:48:52.364+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-06-18T12:49:49.481+0400] {processor.py:161} INFO - Started process (PID=48696) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:49:49.483+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:49:49.484+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:49:49.484+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:49:49.486+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:49:49.486+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:49:49.486+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:49:49.486+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:49:49.487+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:49:49.492+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-06-18T12:50:30.642+0400] {processor.py:161} INFO - Started process (PID=48826) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:50:30.644+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:50:30.645+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:50:30.645+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:50:30.647+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:50:30.646+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:50:30.647+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:50:30.647+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:50:30.647+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:50:30.652+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-06-18T12:51:28.443+0400] {processor.py:161} INFO - Started process (PID=49040) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:51:28.445+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:51:28.446+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:51:28.446+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:51:28.448+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:51:28.448+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:51:28.448+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:51:28.448+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:51:28.448+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:51:28.454+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-06-18T12:52:27.327+0400] {processor.py:161} INFO - Started process (PID=49215) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:52:27.330+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:52:27.331+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:52:27.331+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:52:27.335+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:52:27.335+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:52:27.335+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:52:27.335+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:52:27.335+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:52:27.342+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-06-18T12:53:25.891+0400] {processor.py:161} INFO - Started process (PID=49500) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:53:25.892+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:53:25.894+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:53:25.894+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:53:25.898+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:53:25.898+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:53:25.898+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:53:25.898+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:53:25.899+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:53:25.907+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-06-18T12:54:23.337+0400] {processor.py:161} INFO - Started process (PID=49685) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:54:23.340+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:54:23.341+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:54:23.341+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:54:23.342+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:54:23.342+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:54:23.343+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:54:23.342+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:54:23.343+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:54:23.348+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T12:55:23.287+0400] {processor.py:161} INFO - Started process (PID=49963) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:55:23.289+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:55:23.290+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:55:23.289+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:55:23.292+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:55:23.292+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:55:23.292+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:55:23.292+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:55:23.293+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:55:23.300+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-06-18T12:56:18.268+0400] {processor.py:161} INFO - Started process (PID=50167) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:56:18.270+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:56:18.271+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:56:18.271+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:56:18.274+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:56:18.274+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:56:18.274+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:56:18.274+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:56:18.274+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:56:18.279+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-06-18T12:56:58.744+0400] {processor.py:161} INFO - Started process (PID=50330) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:56:58.746+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:56:58.747+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:56:58.747+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:56:58.748+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:56:58.748+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:56:58.748+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:56:58.748+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:56:58.748+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:56:58.753+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-06-18T12:57:39.287+0400] {processor.py:161} INFO - Started process (PID=50532) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:57:39.289+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:57:39.290+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:57:39.290+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:57:39.293+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:57:39.292+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:57:39.293+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:57:39.293+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:57:39.293+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:57:39.301+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-06-18T12:58:18.690+0400] {processor.py:161} INFO - Started process (PID=50899) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:58:18.692+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:58:18.693+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:58:18.693+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:58:18.695+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:58:18.695+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:58:18.696+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:58:18.696+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:58:18.696+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:58:18.701+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T12:59:17.816+0400] {processor.py:161} INFO - Started process (PID=51301) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:59:17.818+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T12:59:17.819+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:59:17.819+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:59:17.820+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:59:17.820+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T12:59:17.821+0400] {logging_mixin.py:188} INFO - [2024-06-18T12:59:17.821+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T12:59:17.821+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T12:59:17.825+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-06-18T13:00:11.302+0400] {processor.py:161} INFO - Started process (PID=51671) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:00:11.303+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:00:11.305+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:00:11.304+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:00:11.311+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:00:11.310+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:00:11.311+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:00:11.311+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:00:11.311+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:00:11.319+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-06-18T13:01:09.585+0400] {processor.py:161} INFO - Started process (PID=51983) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:01:09.586+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:01:09.588+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:01:09.587+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:01:09.590+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:01:09.590+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:01:09.591+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:01:09.591+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:01:09.591+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:01:09.597+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-06-18T13:02:09.567+0400] {processor.py:161} INFO - Started process (PID=52769) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:02:09.569+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:02:09.570+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:02:09.570+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:02:09.572+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:02:09.572+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:02:09.572+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:02:09.572+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:02:09.572+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:02:09.579+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T13:03:07.815+0400] {processor.py:161} INFO - Started process (PID=53042) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:03:07.817+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:03:07.817+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:03:07.817+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:03:07.819+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:03:07.819+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:03:07.819+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:03:07.819+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:03:07.819+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:03:07.825+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T13:04:04.453+0400] {processor.py:161} INFO - Started process (PID=53243) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:04:04.455+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:04:04.457+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:04:04.456+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:04:04.459+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:04:04.458+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:04:04.459+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:04:04.459+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:04:04.459+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:04:04.465+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-06-18T13:05:04.519+0400] {processor.py:161} INFO - Started process (PID=53482) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:05:04.521+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:05:04.522+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:05:04.522+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:05:04.524+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:05:04.524+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:05:04.525+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:05:04.525+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:05:04.525+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:05:04.530+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T13:06:01.327+0400] {processor.py:161} INFO - Started process (PID=53699) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:06:01.329+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:06:01.330+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:06:01.329+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:06:01.332+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:06:01.332+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:06:01.332+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:06:01.332+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:06:01.332+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:06:01.341+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-06-18T13:07:02.412+0400] {processor.py:161} INFO - Started process (PID=53921) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:07:02.415+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:07:02.415+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:07:02.415+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:07:02.417+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:07:02.417+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:07:02.417+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:07:02.417+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:07:02.417+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:07:02.422+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-06-18T13:07:43.760+0400] {processor.py:161} INFO - Started process (PID=54089) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:07:43.763+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:07:43.764+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:07:43.764+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:07:43.767+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:07:43.767+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:07:43.767+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:07:43.767+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:07:43.768+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:07:43.773+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-06-18T13:08:23.815+0400] {processor.py:161} INFO - Started process (PID=54237) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:08:23.818+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:08:23.819+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:08:23.818+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:08:23.821+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:08:23.821+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:08:23.821+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:08:23.821+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:08:23.821+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:08:23.826+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T13:09:20.798+0400] {processor.py:161} INFO - Started process (PID=54478) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:09:20.800+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:09:20.801+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:09:20.801+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:09:20.803+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:09:20.803+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:09:20.803+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:09:20.803+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:09:20.803+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:09:20.809+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T13:10:02.377+0400] {processor.py:161} INFO - Started process (PID=54621) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:10:02.379+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:10:02.381+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:10:02.381+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:10:02.384+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:10:02.384+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:10:02.385+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:10:02.385+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:10:02.385+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:10:02.390+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-06-18T13:10:44.072+0400] {processor.py:161} INFO - Started process (PID=54775) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:10:44.072+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:10:44.074+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:10:44.073+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:10:44.078+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:10:44.077+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:10:44.078+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:10:44.078+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:10:44.078+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:10:44.083+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-06-18T13:11:25.152+0400] {processor.py:161} INFO - Started process (PID=54916) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:11:25.152+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:11:25.153+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:11:25.153+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:11:25.155+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:11:25.154+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:11:25.155+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:11:25.155+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:11:25.155+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:11:25.160+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-06-18T13:12:22.937+0400] {processor.py:161} INFO - Started process (PID=55102) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:12:22.939+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:12:22.940+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:12:22.940+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:12:22.941+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:12:22.941+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:12:22.942+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:12:22.942+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:12:22.942+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:12:22.947+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-06-18T13:13:22.704+0400] {processor.py:161} INFO - Started process (PID=55342) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:13:22.706+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:13:22.707+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:13:22.707+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:13:22.708+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:13:22.708+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:13:22.709+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:13:22.709+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:13:22.709+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:13:22.714+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T13:14:03.265+0400] {processor.py:161} INFO - Started process (PID=55482) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:14:03.268+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:14:03.270+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:14:03.269+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:14:03.272+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:14:03.272+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:14:03.272+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:14:03.272+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:14:03.273+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:14:03.278+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-06-18T13:15:00.789+0400] {processor.py:161} INFO - Started process (PID=55689) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:15:00.791+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:15:00.792+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:15:00.792+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:15:00.795+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:15:00.795+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:15:00.795+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:15:00.795+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:15:00.795+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:15:00.801+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-06-18T13:15:58.917+0400] {processor.py:161} INFO - Started process (PID=55896) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:15:58.919+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:15:58.920+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:15:58.920+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:15:58.921+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:15:58.921+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:15:58.922+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:15:58.921+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:15:58.922+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:15:58.927+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-06-18T13:16:40.799+0400] {processor.py:161} INFO - Started process (PID=56034) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:16:40.801+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:16:40.802+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:16:40.802+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:16:40.804+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:16:40.803+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:16:40.804+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:16:40.804+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:16:40.804+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:16:40.809+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-06-18T13:17:34.739+0400] {processor.py:161} INFO - Started process (PID=56215) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:17:34.741+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:17:34.742+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:17:34.742+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:17:34.745+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:17:34.744+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:17:34.745+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:17:34.745+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:17:34.745+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:17:34.751+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-06-18T13:18:35.593+0400] {processor.py:161} INFO - Started process (PID=56460) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:18:35.595+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:18:35.596+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:18:35.595+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:18:35.597+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:18:35.597+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:18:35.597+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:18:35.597+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:18:35.597+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:18:35.602+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-06-18T13:19:33.665+0400] {processor.py:161} INFO - Started process (PID=56659) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:19:33.668+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:19:33.670+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:19:33.669+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:19:33.673+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:19:33.673+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:19:33.673+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:19:33.673+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:19:33.674+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:19:33.679+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-06-18T13:20:32.510+0400] {processor.py:161} INFO - Started process (PID=56866) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:20:32.513+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:20:32.514+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:20:32.514+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:20:32.516+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:20:32.516+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:20:32.516+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:20:32.516+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:20:32.516+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:20:32.521+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-06-18T13:21:30.928+0400] {processor.py:161} INFO - Started process (PID=57066) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:21:30.930+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:21:30.931+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:21:30.930+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:21:30.932+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:21:30.932+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:21:30.932+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:21:30.932+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:21:30.932+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:21:30.938+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-06-18T13:22:12.793+0400] {processor.py:161} INFO - Started process (PID=57208) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:22:12.795+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:22:12.796+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:22:12.796+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:22:12.798+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:22:12.798+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:22:12.798+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:22:12.798+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:22:12.798+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:22:12.804+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T13:23:10.880+0400] {processor.py:161} INFO - Started process (PID=57419) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:23:10.882+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:23:10.884+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:23:10.883+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:23:10.886+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:23:10.886+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:23:10.886+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:23:10.886+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:23:10.886+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:23:10.891+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T13:24:09.720+0400] {processor.py:161} INFO - Started process (PID=57629) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:24:09.722+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:24:09.723+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:24:09.723+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:24:09.725+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:24:09.725+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:24:09.726+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:24:09.726+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:24:09.726+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:24:09.731+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-06-18T13:25:07.517+0400] {processor.py:161} INFO - Started process (PID=57827) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:25:07.519+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:25:07.520+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:25:07.519+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:25:07.521+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:25:07.521+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:25:07.521+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:25:07.521+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:25:07.521+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:25:07.527+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-06-18T13:26:05.737+0400] {processor.py:161} INFO - Started process (PID=58023) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:26:05.739+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:26:05.740+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:26:05.740+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:26:05.742+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:26:05.742+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:26:05.742+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:26:05.742+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:26:05.742+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:26:05.747+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-06-18T13:27:01.888+0400] {processor.py:161} INFO - Started process (PID=58214) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:27:01.889+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:27:01.890+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:27:01.890+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:27:01.892+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:27:01.892+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:27:01.892+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:27:01.892+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:27:01.892+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:27:01.898+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-06-18T13:27:57.762+0400] {processor.py:161} INFO - Started process (PID=58448) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:27:57.764+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:27:57.765+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:27:57.765+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:27:57.767+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:27:57.767+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:27:57.768+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:27:57.768+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:27:57.768+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:27:57.773+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T13:28:38.957+0400] {processor.py:161} INFO - Started process (PID=58587) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:28:38.958+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:28:38.959+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:28:38.959+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:28:38.961+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:28:38.961+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:28:38.961+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:28:38.961+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:28:38.961+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:28:38.966+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-06-18T13:29:33.741+0400] {processor.py:161} INFO - Started process (PID=58772) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:29:33.743+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:29:33.745+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:29:33.744+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:29:33.747+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:29:33.746+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:29:33.747+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:29:33.747+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:29:33.747+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:29:33.752+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-06-18T13:30:33.016+0400] {processor.py:161} INFO - Started process (PID=59055) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:30:33.018+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:30:33.020+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:30:33.019+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:30:33.022+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:30:33.021+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:30:33.022+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:30:33.022+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:30:33.022+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:30:33.028+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-06-18T13:31:31.757+0400] {processor.py:161} INFO - Started process (PID=59290) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:31:31.759+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:31:31.760+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:31:31.760+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:31:31.762+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:31:31.762+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:31:31.762+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:31:31.762+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:31:31.762+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:31:31.767+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-06-18T13:32:15.689+0400] {processor.py:161} INFO - Started process (PID=59611) to work on /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:32:15.689+0400] {processor.py:830} INFO - Processing file /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T13:32:15.690+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:32:15.690+0400] {dagbag.py:545} INFO - Filling up the DagBag from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:32:15.692+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:32:15.691+0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T13:32:15.692+0400] {logging_mixin.py:188} INFO - [2024-06-18T13:32:15.692+0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T13:32:15.692+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T13:32:15.698+0400] {processor.py:183} INFO - Processing /home/user/Desktop/airflowproj/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
